sympy/functions/elementary/tests/test_hyperbolic.py

```diff
@@ -454,21 +454,24 @@ def test_asinh_series():
 
 
 def test_acosh():
-    # TODO please write more tests  -- see issue 3751
-    # From http://functions.wolfram.com/ElementaryFunctions/ArcCosh/03/01/
-    # at specific points
     x = Symbol('x')
 
     assert acosh(-x) == acosh(-x)
 
+    #at specific points
     assert acosh(1) == 0
     assert acosh(-1) == pi*I
     assert acosh(0) == I*pi/2
     assert acosh(Rational(1, 2)) == I*pi/3
     assert acosh(Rational(-1, 2)) == 2*pi*I/3
 
+    # at infinites
+    assert acosh(oo) == oo
+    assert acosh(-oo) == oo
+
     assert acosh(zoo) == oo
 
+    #properties
```

Comment:
Such are usually referred to as `simplify`, that can go in the comment. Either way is fine by me.